aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsCodegenDecorator.kt

```diff
@@ -13,17 +14,20 @@ import software.amazon.smithy.rustsdk.customize.s3.S3Decorator
 
 val DECORATORS = listOf(
     // General AWS Decorators
-    CredentialsProviderDecorator(),
-    RegionDecorator(),
     AwsEndpointDecorator(),
-    UserAgentDecorator(),
-    SigV4SigningDecorator(),
-    RetryPolicyDecorator(),
-    IntegrationTestDecorator(),
     AwsFluentClientDecorator(),
+    AwsPresigningDecorator(),
     CrateLicenseDecorator(),
+    CredentialsProviderDecorator(),
+    IntegrationTestDecorator(),
+    RegionDecorator(),
```

Comment:
the order here _shouldn't_ matter, but do you mind looking at a generated code diff just to make sure it doesn't introduce a huge diff?

Reply:
This change does re-order the declaration of internal methods in generated code, I'll remove it and make a separate PR for sorting this list